main.tf

```diff
@@ -2,6 +2,12 @@ locals {
   #Repo attributes that are meant to simplify configuration rather than being actual repo options
   helper_options = ["allow_all_server_side_workflows", "terragrunt_atlantis_config", "infracost"]
 
+  infracost_parameters = {
+    github : { token_name : "github-token", environment_variable_name : "GITHUB_TOKEN" },
```

Comment:
Was thinking to to change the variable names to be more specific `INFRACOST_GITHUB_TOKEN` etc. `GITHUB_TOKEN` is too generic and if we want to be more granual with permissions, we could then pass token with permissions to only post comments.

Reply:
yeah, you're right

(I was thinking about moving that `environment_variable_name` into variables (with default) but was worried about too many knobs there)

Reply:
yeah, very little value in exchange for possible technical debt